src/Agent/NewRelic/Profiler/Common/CorStandIn.h

```diff
@@ -3,6 +3,7 @@
 * SPDX-License-Identifier: Apache-2.0
 */
 #pragma warning(push)
-#pragma warning(disable : 4458)
+#pragma warning(disable: 4458)
```

Comment:
Could you add a comment to describe the warning that's being disabled?

Reply:
Thanks, I meant to do that but forgot.